hs_core/hydroshare/users.py

```diff
@@ -332,13 +332,16 @@ def get_resource_list(creator=None, group=None, user=None, owner=None, from_date
                 coverages.add(coverage.id)
 
         for coverage in Coverage.objects.filter(type="point"):
-            coverage_shape = Point(
-                coverage.value.get('east', None),
-                coverage.value.get('north', None),
-            )
-
-            if search_polygon.intersects(coverage_shape):
-                coverages.add(coverage.id)
+            try:
+                coverage_shape = Point(
+                    coverage.value.get('east', None),
+                    coverage.value.get('north', None),
+                )
+
+                if search_polygon.intersects(coverage_shape):
+                    coverages.add(coverage.id)
+            except Exception as e:
+                pass
```

Comment:
Check this file https://github.com/hydroshare/hydroshare/blob/a31f2d6fded728dc341bb1ad46a65ddcfdae6511/hs_file_types/models/raster.py for an example of using logging. Basically, you can call ```log = logging.getLogger(__name__)``` then call ```log.info```, ```log.debug```, etc., to record logging messages.